packages/databricks-vscode/src/utils/onErrorDecorator.ts

```diff
@@ -0,0 +1,67 @@
+import {window} from "vscode";
+import {
+    getContextParamIndex,
+    logging,
+    Context,
+} from "@databricks/databricks-sdk";
+
+interface Props {
+    popup?:
+        | {
+              prefix?: string;
+          }
+        | true;
+    log?: {
+        logger: string;
+        prefix?: string;
+    };
```

Comment:
Do we expect to supply different loggers here? Can we simplify for now and always use `Loggers.Extension`? We can always add the log parameter later if we need it

Reply:
Updated